draft-irtf-cfrg-vdaf.md

```diff
@@ -3480,6 +3544,21 @@ on the set of all aggregation parameters used for a Client's input share, and
 only proceed with the preparation and aggregation phases if that function call
 returns `True`.
 
+## Pseudorandom Generators and Random Oracles
+
+The objects we describe in {{prg}} share a common interface, which we have
+called Prg. However, these are not necessarily all modeled as cryptographic
+Pseudorandom Generators in the security analyses of our protocols. Instead, most
+of them are modeled as Random Oracles. For these use cases, we want to be
+conservative in our assumptions, and hence prescribe PrgSha3 as the only
+RECOMMENDED Prg instantiation.
+
+The one exception is the PRG used in the Idpf implementation IdpfPoplar
+{{idpf-poplar}}. Here, a Random Oracle is not needed to prove security, and
+hence a construction based on fixed-key AES {{prg-fixed-key-aes-128}} can be
+used. However, as PrgFixedKeyAes128 has been shown to be differentiable from
+a Random Oracle {{GKWWY20}}, it is NOT RECOMMENDED to use it anywhere else.
```

Comment:
The end of this section should point to the OPEN ISSUE of #159.

Reply:
Done.